paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -378,6 +378,12 @@ public class CoreOptions implements Serializable {
                     .defaultValue(MemorySize.ofMebiBytes(128))
                     .withDescription("Target size of a file.");
 
+    public static final ConfigOption<Boolean> FORCE_EXPIRE =
```

Comment:
Maybe it is better to introduce an option: write.responsible-matters, similar option, I'm not good at naming it, for reference only.

In the option, user can specific what the write does.